Units/parser-javascript.r/js-es6-mixin.d/input.js

```diff
@@ -0,0 +1,28 @@
+
+class Foo {
+  foo() { return "foo"; }
+}
+
+class Bar extends Foo {
+  bar() { return "bar"; }
+}
+
+function Mixin(superclass) {
+  return class extends superclass {
+    hello() {
```

Comment:
Just a question. `hello` method(?) in the anonymous class is not captured. Is this expected behaviour?